specs/MultiProfile.md

```diff
@@ -328,6 +362,18 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
+  /// After the Delete() call completes, The render process of webviews on this profile will
+  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
+  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
+  /// for more details. The profile directory on disk will be actually deleted when the browser
```

Comment:
Regarding 'The profile folder will be deleted when the browser process exits'. Does this mean that if you have one browser process with two profiles and a CoreWebView2 running in each profile, and you delete one profile, the deleted profile's profile folder on disk is not deleted until the other CoreWebView2 closes down and the browser process exits? Or is it when all the CoreWebView2s from a profile close down that the profile folder is deleted?

Reply:
Is browser process exits. In fact, it will try to delete local file immediately, but the local file will always be occupied.

Reply:
The local file delete logic is:

1.  Try to delete immediately when the `profile_manager()->MaybeScheduleProfileForDeletion() `  called.
2.  Try to delete the file which is marked as deleted at browser process exit.
3.  Ensure the marked file is deleted completely, if not, try to delete it again.